tiledb/sm/c_api/tiledb_experimental.h

```diff
@@ -454,6 +454,34 @@ TILEDB_EXPORT int32_t tiledb_query_condition_set_use_enumeration(
     const tiledb_query_condition_t* cond,
     int use_enumeration) TILEDB_NOEXCEPT;
 
+/* ********************************* */
+/*           QUERY PREDICATE         */
+/* ********************************* */
+
+/**
+ * Adds a predicate to be applied to a read query. The added predicate
+ * will be analyzed and evaluated in the subarray step, query condition
+ * step, or both.
+ *
+ * The predicate is parsed as a SQL expression and must evaluate
```

Comment:
```suggestion
 * The predicate is parsed as a DataFusion SQL expression and must
 * evaluate
```

We should mention the SQL dialect as a temporary measure until we specify it in our own documentation (issue TBA).

Reply:
653c8907